File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/injs/InjsINJ1001U01TempListItemInfo.java
package nta.med.data.model.ihis.injs;

import java.sql.Timestamp;

public class InjsINJ1001U01TempListItemInfo {
	private String serialV;
	private String bunho;
	private String suname;
	private String suname2;
	private Integer age;
	private String sex;
	private Timestamp orderDate;
	private Timestamp jubsuDate;
	private Double cnt;
	private Double suryang;
	private String danuiName;
	private String bogyongName;
	private String jusaName;
	private String jaeryoName;
	private String orderRemark;
	private String dataGubun;
	private String birth;
	private String doctorName;
	private String gwaName;
	public InjsINJ1001U01TempListItemInfo(String serialV, String bunho,
			String suname, String suname2, Integer age, String sex,
			Timestamp orderDate, Timestamp jubsuDate, Double cnt,
			Double suryang, String danuiName, String bogyongName,
			String jusaName, String jaeryoName, String orderRemark,
			String dataGubun, String birth, String doctorName, String gwaName) {
		super();
		this.serialV = serialV;
		this.bunho = bunho;
		this.suname = suname;
		this.suname2 = suname2;
		this.age = age;
		this.sex = sex;
		this.orderDate = orderDate;
		this.jubsuDate = jubsuDate;
		this.cnt = cnt;
		this.suryang = suryang;
		this.danuiName = danuiName;
		this.bogyongName = bogyongName;
		this.jusaName = jusaName;
		this.jaeryoName = jaeryoName;
		this.orderRemark = orderRemark;
		this.dataGubun = dataGubun;
		this.birth = birth;
		this.doctorName = doctorName;
		this.gwaName = gwaName;
	}
	public String getSerialV() {
		return serialV;
	}
	public void setSerialV(String serialV) {
		this.serialV = serialV;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getSuname2() {
		return suname2;
	}
	public void setSuname2(String suname2) {
		this.suname2 = suname2;
	}
	public Integer getAge() {
		return age;
	}
	public void setAge(Integer age) {
		this.age = age;
	}
	public String getSex() {
		return sex;
	}
	public void setSex(String sex) {
		this.sex = sex;
	}
	public Timestamp getOrderDate() {
		return orderDate;
	}
	public void setOrderDate(Timestamp orderDate) {
		this.orderDate = orderDate;
	}
	public Timestamp getJubsuDate() {
		return jubsuDate;
	}
	public void setJubsuDate(Timestamp jubsuDate) {
		this.jubsuDate = jubsuDate;
	}
	public Double getCnt() {
		return cnt;
	}
	public void setCnt(Double cnt) {
		this.cnt = cnt;
	}
	public Double getSuryang() {
		return suryang;
	}
	public void setSuryang(Double suryang) {
		this.suryang = suryang;
	}
	public String getDanuiName() {
		return danuiName;
	}
	public void setDanuiName(String danuiName) {
		this.danuiName = danuiName;
	}
	public String getBogyongName() {
		return bogyongName;
	}
	public void setBogyongName(String bogyongName) {
		this.bogyongName = bogyongName;
	}
	public String getJusaName() {
		return jusaName;
	}
	public void setJusaName(String jusaName) {
		this.jusaName = jusaName;
	}
	public String getJaeryoName() {
		return jaeryoName;
	}
	public void setJaeryoName(String jaeryoName) {
		this.jaeryoName = jaeryoName;
	}
	public String getOrderRemark() {
		return orderRemark;
	}
	public void setOrderRemark(String orderRemark) {
		this.orderRemark = orderRemark;
	}
	public String getDataGubun() {
		return dataGubun;
	}
	public void setDataGubun(String dataGubun) {
		this.dataGubun = dataGubun;
	}
	public String getBirth() {
		return birth;
	}
	public void setBirth(String birth) {
		this.birth = birth;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	@Override
	public String toString() {
		return "InjsINJ1001U01TempListItemInfo [serialV=" + serialV
				+ ", bunho=" + bunho + ", suname=" + suname + ", suname2="
				+ suname2 + ", age=" + age + ", sex=" + sex + ", orderDate="
				+ orderDate + ", jubsuDate=" + jubsuDate + ", cnt=" + cnt
				+ ", suryang=" + suryang + ", danuiName=" + danuiName
				+ ", bogyongName=" + bogyongName + ", jusaName=" + jusaName
				+ ", jaeryoName=" + jaeryoName + ", orderRemark=" + orderRemark
				+ ", dataGubun=" + dataGubun + ", birth=" + birth
				+ ", doctorName=" + doctorName + ", gwaName=" + gwaName + "]";
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Injs/InjsINJ1001U01TempListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Injs
{
    [Serializable]
	public class InjsINJ1001U01TempListItemInfo
	{
		private String _serialV;
		private String _bunho;
		private String _suname;
		private String _suname2;
		private String _age;
		private String _sex;
		private String _orderDate;
		private String _jubsuDate;
		private String _cnt;
		private String _suryang;
		private String _danuiName;
		private String _bogyongName;
		private String _jusaName;
		private String _jaeryoName;
		private String _orderRemark;
		private String _dataGubun;
		private String _birth;
		private String _doctorName;
		private String _gwaName;

		public String SerialV
		{
			get { return this._serialV; }
			set { this._serialV = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Suname
		{
			get { return this._suname; }
			set { this._suname = value; }
		}

		public String Suname2
		{
			get { return this._suname2; }
			set { this._suname2 = value; }
		}

		public String Age
		{
			get { return this._age; }
			set { this._age = value; }
		}

		public String Sex
		{
			get { return this._sex; }
			set { this._sex = value; }
		}

		public String OrderDate
		{
			get { return this._orderDate; }
			set { this._orderDate = value; }
		}

		public String JubsuDate
		{
			get { return this._jubsuDate; }
			set { this._jubsuDate = value; }
		}

		public String Cnt
		{
			get { return this._cnt; }
			set { this._cnt = value; }
		}

		public String Suryang
		{
			get { return this._suryang; }
			set { this._suryang = value; }
		}

		public String DanuiName
		{
			get { return this._danuiName; }
			set { this._danuiName = value; }
		}

		public String BogyongName
		{
			get { return this._bogyongName; }
			set { this._bogyongName = value; }
		}

		public String JusaName
		{
			get { return this._jusaName; }
			set { this._jusaName = value; }
		}

		public String JaeryoName
		{
			get { return this._jaeryoName; }
			set { this._jaeryoName = value; }
		}

		public String OrderRemark
		{
			get { return this._orderRemark; }
			set { this._orderRemark = value; }
		}

		public String DataGubun
		{
			get { return this._dataGubun; }
			set { this._dataGubun = value; }
		}

		public String Birth
		{
			get { return this._birth; }
			set { this._birth = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String GwaName
		{
			get { return this._gwaName; }
			set { this._gwaName = value; }
		}

		public InjsINJ1001U01TempListItemInfo() { }

		public InjsINJ1001U01TempListItemInfo(String serialV, String bunho, String suname, String suname2, String age, String sex, String orderDate, String jubsuDate, String cnt, String suryang, String danuiName, String bogyongName, String jusaName, String jaeryoName, String orderRemark, String dataGubun, String birth, String doctorName, String gwaName)
		{
			this._serialV = serialV;
			this._bunho = bunho;
			this._suname = suname;
			this._suname2 = suname2;
			this._age = age;
			this._sex = sex;
			this._orderDate = orderDate;
			this._jubsuDate = jubsuDate;
			this._cnt = cnt;
			this._suryang = suryang;
			this._danuiName = danuiName;
			this._bogyongName = bogyongName;
			this._jusaName = jusaName;
			this._jaeryoName = jaeryoName;
			this._orderRemark = orderRemark;
			this._dataGubun = dataGubun;
			this._birth = birth;
			this._doctorName = doctorName;
			this._gwaName = gwaName;
		}

	}
}